ilmsdump/gui/__init__.py

```diff
@@ -0,0 +1,119 @@
+import asyncio
+import sys
+from datetime import datetime
+
+from PySide6.QtCore import QCoreApplication, QFile, QObject, Qt, QThread
+from PySide6.QtUiTools import QUiLoader
+from PySide6.QtWidgets import QApplication, QLineEdit, QPushButton, QTextBrowser
+
+import ilmsdump
+
+target_path = '.'
+
+
+def addTextBrowser(text_browser, text):
+    text_browser.setText(text_browser.toPlainText() + text)
+
+
+async def get_enrolled_courses(client, log):
+    courses = []
+    try:
+        await [courses.append(i) async for i in client.get_enrolled_courses()]
+    except TypeError:
+        pass
```

Comment:
TypeError 是 `TypeError: object list can't be used in 'await' expression` 嗎？
```suggestion
    courses = [c async for c in client.get_enrolled_courses()]
```

Reply:
沒錯，被抓包XD，我沒詳細去看原因，一個 workaround

Reply:
fixed